cli/hzn.go

```diff
@@ -566,6 +567,9 @@ Environment Variables:
 	mmsObjectPublishPat := mmsObjectPublishCmd.Flag("pattern", msgPrinter.Sprintf("If you want the object to be deployed on nodes using a given pattern, specify it using this flag. This flag is optional and can only be used with --type and --id. It is mutually exclusive with -m")).Short('p').String()
 	mmsObjectPublishDef := mmsObjectPublishCmd.Flag("def", msgPrinter.Sprintf("The definition of the object to publish. A blank template can be obtained from the 'hzn mss object new' command. Specify -m- to read from stdin.")).Short('m').String()
 	mmsObjectPublishObj := mmsObjectPublishCmd.Flag("object", msgPrinter.Sprintf("The object (in the form of a file) to publish. This flag is optional so that you can update only the object's definition.")).Short('f').String()
+	mmsObjectPublishSkipDigitalSig := mmsObjectPublishCmd.Flag("skipDigitalSig", msgPrinter.Sprintf("Indicate no data signature applied to the object data (the file passed in with -f flag). It is mutually exclusive with -s")).Bool()
+	mmsObjectPublishDSHashAlgo := mmsObjectPublishCmd.Flag("hashAlgo", msgPrinter.Sprintf("The hash algorithm used for hash generation and digital signing. Supported hash algorithms are SHA1, SHA256. Default is SHA1. It is mutually exclusive with --skipDigitalSig")).Short('a').String()
+	mmsObjectPublishDSHash := mmsObjectPublishCmd.Flag("hash", msgPrinter.Sprintf("The hash value to digital sign the object data (the file passed in with -f flag). This string should be SHA1 or SHA256 (specify with -a flag) hash value encoded in hexadecimal. It is mutually exclusive with --skipDigitalSig")).Short('s').String()
```

Comment:
The -s flag is used in other places in the CLI. I think we should have no short flag for --hash, since it wont be used very often. Also, let's change the description: "The hash of the object data being uploaded or downloaded. Use this flag if you want to provide the hash instead of allowing the command to automatically calculate the hash. The hash must be generated using either the SHA1 or SHA256 algorithm. The -a flag must be specified if the hash was generated using SHA256. This flag is mutually exclusive with --noIntegrity."